src/Modulo2.jl

```diff
@@ -84,6 +84,7 @@ isodd(a::ZZ2) = isone(a)
 
 +(a::ZZ2) = a
 +(a::ZZ2, b::ZZ2) = ZZ2(xor(a.m, b.m))
+xor(a::ZZ2, b::ZZ2) = a + b # for syntactic consistency
```

Comment:
I'm sceptical about this. If you want to use `xor`, then you probably want to use `Bool` instead of `ZZ2`. (And why only `xor` and not `&` for multiplication?)

Reply:
I in fact think `&` for multiplication should also be done! (I hadn't added it yet because I have not yet needed that implementation in my separate usecase). Regarding the original addition: I think it is justified because: (i) sometimes one may wish to be explicit to the reader (of code) that the action is in practice `xor`ing. (ii) To switch types to `Bool` or `BitArray` would be a hassle. (iii) In my real use case, I have functions with generic methods that act perform `xor` elementwise between two vectors; sometimes I pass to them `ZZ2Vector`, sometime `Vector{Int64}`, sometimes `BitArray`. It would be helpful for a genetic method that invokes `xor` to just work on `ZZ2Vector` as well.